docker/images/pinot-base/pinot-base-build/openjdk.dockerfile

```diff
@@ -16,33 +16,37 @@
 # specific language governing permissions and limitations
 # under the License.
 #
-ARG JAVA_VERSION=11
+ARG JAVA_VERSION=21
```

Comment:
No. We are using this to test 21 and later once jdk21 is GA, we can just use it in prod.

Also, I think we should just go with java LTS versions, unless asked.

Reply:
:+1:

Reply:
As an extra, we can periodically (maybe daily is too much, but once per month or once per Q) we can compile and run tests with the latest GA version in order to check whether new versions break something in order to be ready to release ASAP when the next LTS is released.

BTW, in case we don't want to use a EA version, we should not use openjdk (as indicated in their own [docker page](https://hub.docker.com/_/openjdk)). The only reason to use it is because it is the only distribution that offers EAs.

Reply:
I think we should publish 21 EA version for pioneer users, and gradually updated the runtime base image to catch up latest java 21 EA changes and  eventually goes to GA version.